### DISCUSSION TITLE
Lost AIO password on Windows Docker install.

### DISCUSSION THREAD

**Discussion by nerlins** [nextcloud/all-in-one#1110] [Questions] [answered]
I have looked around for the answer. I know I can 'grep password' on the AIO container, but I do not know how to run the command properly. All commands are Linux derivative, directing me to look in /var. Can someone educate me on on where to look for the configuration.json, when running terminal commands on Windows? I have tried Command Prompt, and the built-in Docker Terminal.

**Comment by nerlins** [comment]
Ok, well I dug around in the Docker GUI, and found the data folder in the container, then was able to download the configuration.json file.  I still don't know why I can't run commands in the Docker terminal, like in this post:

https://github.com/nextcloud/all-in-one/discussions/803#discussioncomment-2905075

Is there something I am missing on my Docker installation?  Maybe I still have some learning to do about Docker on Windows...This is a lot more awkward than on Debian, but this Windows server is all I have for Nextcloud right now.

**Reply by Zoey2936** [reply]
The comment you sent is only for Linux

**Reply by Zoey2936** [reply]
If you open the configuration.json simply search for `"password":"123",` 123 would be your password in this example

**Reply by nerlins** [reply]
Well, how would it work then? If I open the built in terminal in Docker, then I can run commands for the container. I have learned, just now, that grep doesn't work on Windows Command Prompt, but does it not work in the Docker terminal?

**Reply by nerlins** [reply]
> If you open the configuration.json simply search for `"password":"123",` 123 would be your password in this example

I did find that. I guess I am now more curious how I can actually run commands I might find in Nextcloud documentation, within the Docker terminal for a specific container. I just cant seem to figure it out. My syntax is incorrect, or maybe I am missing a plugin. Just not sure.

**Reply by Zoey2936** [reply]
You can try this command, it should also work on windows:
`docker exec nextcloud-aio-mastercontainer cat /mnt/docker-aio-config/data/configuration.json`

**Reply by nerlins** [reply]
> You can try this command, it should also work on windows: `docker exec nextcloud-aio-mastercontainer cat /mnt/docker-aio-config/data/configuration.json | grep password`

Oh, wow. That actually worked, BUT, I had to replace grep with findstr. Ok...so this has definitely helped me visualize the syntax I need to run terminal commands for Docker Windows. I'm just so used to Debian. Thank you very much!

**Comment by Zoey2936** [comment, marked as answer]
This command seems to work (also on Windows):
`docker exec nextcloud-aio-mastercontainer cat /mnt/docker-aio-config/data/configuration.json`
Then search for `"password":"123",` then 123 would be your password in this example

**Reply by nerlins** [reply]
Yep. That works as well. :-)
